projects/cds/src/profiletag/model/profile-tag.model.ts

```diff
@@ -17,5 +17,12 @@ export interface ProfileTagJsConfig {
 }
 
 export interface ProfileTagEvent {
-  eventName: string;
+  name: ProfileTagEventNames;
+  data: any;
```

Comment:
I don't see where `data` is populated?
And is there a type for it, that it can used?

Reply:
This is created by our profiletag.js. This is removed in another branch as I refactored the events to instead come from the window: https://github.com/SAP/cloud-commerce-spartacus-storefront/blob/feature/ARGO-refactoring/projects/cds/src/profiletag/model/profile-tag.model.ts